clusters/privatek8s.yaml

```diff
@@ -43,6 +46,34 @@ releases:
       - "../config/ext_datadog_privatek8s.yaml"
     secrets:
       - "../secrets/config/datadog/privatek8s-secrets.yaml"
+  - name: jenkins-infra
+    namespace: jenkins-infra
+    chart: jenkins/jenkins
+    version: 4.2.17
+    needs:
+      - jenkins-infra-additional # Required to allow the Jenkins controller to spawn kubernetes pods in the current cluster
```

Comment:
I'm not sure to understand the meaning of this comment. Is this a copy and paste from the current infra.ci (temp-privatek8s) manifest?

Reply:
It's replicated from the current infra.ci: https://github.com/jenkins-infra/kubernetes-management/blob/main/clusters/temp-privatek8s.yaml#L53-L54